tests/model_serving/model_server/kserve/ingress/conftest.py

```
@@ -23,56 +22,74 @@
             }
         ):
-            LOGGER.info(f"Wait for inference service {s3_models_inference_service.name} url update")
+            LOGGER.info(f"Wait for inference service {ovms_kserve_inference_service.name} url update")
             for sample in TimeoutSampler(
                 wait_timeout=2 * 60,
                 sleep=1,
-                func=lambda: s3_models_inference_service.instance.status.url,
+                func=lambda: ovms_kserve_inference_service.instance.status.url,
             ):
                 if sample:  # noqa: SIM102
                     if visibility == Labels.Kserve.EXPOSED and isvc_orig_url == sample or sample != isvc_orig_url:
```

Ruff (0.15.12)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)